plasma_framework/test/src/exits/payment/PaymentProcessInFlightExit.test.js

```diff
@@ -0,0 +1,446 @@
+/* eslint no-bitwise: ["error", { "allow": ["|"] }] */
```

Comment:
How about a scenario that checks if we do not mark to many inputs as spent?
Basically let's make an input exit where not all inputs for a token are piggybacked.
I have a feeling those not piggybacked inputs would be mark as spent in current code.

Reply:
Your feeling is correct and it is intended actually.

we do have such test case here actually. 
see the test: `should flag all inputs with the same token as spent` and the setup in the `beforeEach` of such test. I will add some comment in that test to clarify _both_ piggybacked and non-piggybacked input would be flagged.